picard/ui/mainwindow/actions.py

```diff
@@ -348,10 +348,11 @@ def _create_show_toolbar_action(parent):
 
 
 @add_action(MainAction.SHOW_FINDBAR)
-def _create_find_toolbar_action(parent):
-    action = QtGui.QAction(_("Find"), parent)
+def _create_find_bar_action(parent):
```

Comment:
To be consistent, it should be named `_create_filter_bar_action()` (see my comment about find -> filter).